cpmonitor/models.py

```diff
@@ -358,13 +358,18 @@ class Meta:
         help_text="Nur falls im Klimaaktionsplan angegeben.",
     )
 
-    responsible_organ = models.TextField(
+    responsible_organ = models.CharField(
         "Verantwortliches Organ",
         blank=True,
+        max_length=100,
```

Comment:
```suggestion
        max_length=100,
        help_text="""
            <p>Kurzer Name (ggf. gebräuchliche Abkürzung) des verantwortlichen Gremiums oder der verantwortlichen Behörde.</p>
        """,
```

Reply:
For me the field name is pretty self-explaining...
And I'd like to save the pilots the time of reading help texts.

Reply:
I find "Organ" quite difficult to grasp for some people. Explaining this might help. Also the suggestion to use an abbreviation is important. (See my "SKUMS" example above.)

And: I think we should put help text on each field. They can be very short, if there really is no relevant information. As to why: I can explain that in person.

Reply:
I re-added the helptexts. Hope its fine now.